addons/binding/org.openhab.binding.xiaomivacuum/.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry kind="lib" path="lib/bcpkix-jdk15on-1.50.jar"/>
```

Comment:
Yes, sure. I just wanted to point it out in case it really works well without. I would have to read up the differences  too, so unfortunately I'm not a good help regarding this question...